.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
     -    id: check-stack-names
     -    id: check-stack-tag-values
          args: [--tag=CostCenter,
-                --file=https://finops-api.sageit.org/tags,
+                --file=https://mips-api.finops.sageit.org/tags,
```

Comment:
the new endpoint doesn't work.  is this dependent on another PR somewhere?  if so can you please reference it?

Reply:
I was expecting this to work after a successful run of aws-deploy, but it looks like the first run of the lambda hit a 500 during the log in and got locked out for an hour. It's working now.